name_institution_data.csv

```diff
@@ -2,3 +2,4 @@ Last Name,First Name,Institutions
 Poorman,Megan,Texas A&M University,Vanderbilt University,University Medical Center Utrecht,National Institute of Standards and Technology
 Test,Alice,inst1,inst2
 Test,Bob,inst1,inst3
+Borisch,Eric,Case Western Reserve University,Stanford Univiersity,Mayo Clinic
```

Comment:
Comments on the pull request can be recorded directly in the pull request, which will persist on GitHub (however these comments are not captured in the git repository itself). For instance, I can point out a typo, and _fix it directly_ in the pull request.